uitools/common.cmake

```diff
@@ -13,6 +13,43 @@
 # limitations under the License.
 set(CPPPATH ${CMAKE_CURRENT_LIST_DIR}/cpp/Esri/ArcGISRuntime/Toolkit)
 
+set(TOOLKITCOMMON_HEADERS
```

Comment:
I think a good practice with cmake is to have a `CMakeLists.txt` for each directory and `add_subdirectory` to add them to the root `CMakeLists.txt`.

Reply:
So does this mean we should replace
```
# Uncomment this line to add the toolkit to your project
# See https://github.com/Esri/arcgis-maps-sdk-toolkit-qt for details
# include(path/to/toolkitcpp.cmake)
```

with

```
# Uncomment this line to add the toolkit to your project
# See https://github.com/Esri/arcgis-maps-sdk-toolkit-qt for details
# add_subdirectory(path/to/CMakeLists.txt /path/to/buildoutput)
```

Reply:
If this is the better workflow then we should use it, however we cannot have two different CMakeLists.txt in the same directory for Qt Quick and Widgets, so we'll have to change things up for that.

Reply:
>So does this mean we should also replace

Yes if that's what we're doing.